docs/tutorials/monorepo_guidance.md

```diff
@@ -0,0 +1,40 @@
+# Configuring commitizen in a monorepo
+
+This tutorial assumes the monorepo layout is designed with multiple components that can be released independently of each other,
+some suggested layouts:
+
+```
+library-a
+  .cz.toml
+library-b
+  .cz.toml
+```
+
+```
+src
+    library-b
+      .cz.toml
+    library-z
+      .cz.toml
+```
+
+Each component will have its own changelog, commits will need to use scopes so only relevant commits are included in the
```

Comment:
What if you are not using conventional commits and you don't have scopes?

Reply:
Sorry, I guess I made an assumption here that everyone uses them. Should I just reword this to reflect that assumption?